packages/mdc-ripple/mdc-ripple.scss

```diff
@@ -35,11 +35,11 @@
   }
 
   &--primary {
-    @include mdc-states-color(primary);
+    @include mdc-states(primary, true);
```

Comment:
Can we use a named param for `true`? E.g.:

```scss
@include mdc-states(primary, $has-nested-focusable-element: true);
```

At some point I'd like to go through all of our Sass mixins and use named params for all non-obvious boolean values.

Reply:
Yeah, this was actually just for testing and I forgot to remove it... and I sort of intentionally broke exactly this rule to try to remind myself to remove it... but instead you reminded me!